samples/bluetooth/opendroneid/README.rst

```diff
@@ -0,0 +1,23 @@
+.. zephyr:code-sample:: bluetooth_opendroneid
+   :name: OpenDroneID
+   :relevant-api: bluetooth
+
+   Advertise ODID Messages using GAP Broadcaster role.
+
+Overview
+********
+
+A simple application demonstrating ODID Message transmission using the GAP Broadcaster role.
```

Comment:
This is a bit obscure for people (like myself) not familiar with opendroneID. Would be nice to add a bit more context as to what it is about. You may also add a link to opendroneID repo.

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/pull/82034